fedora_bot.py

```diff
@@ -253,11 +297,15 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-u", "--user", help="Set the username of the Fedora account")
     parser.add_argument("-p", "--password", help="Set the Fedora account password")
+    parser.add_argument("-a", "--apikey", help="Set the Fedora account API key")
```

Comment:
I would even be considering omitting `-a`, it just feels like an `--all`  argument.

Reply:
Yeah, I considered dropping the shorthands for everything, that's why I moved to the long names in CI as well.